js/hp-2019/more-links-popover.jquery.js

```diff
@@ -107,6 +109,13 @@ $.fn.moreLinksPopover = function initPlugin(o = {}) {
     }));
 
     $trigger.data('id7.more-links-popover', moreLinksPopover);
+    const $html = $('html');
+    $html.on('keyup', '.megamenu-links', (event) => {
```

Comment:
Should also be in MoreLinksPopover.wireEventHandlers, next to where `click.id7.homepage.popoverDismiss` is defined. Rather than hooking a key event into a mouse event, maybe just make them call a common function.

I don't think this one particularly needs to listen to all keys on `html`, it could just listen directly to keydowns on the popover element whenever it's shown (and unbind when hidden)